http/wsclient.go

```diff
@@ -398,24 +471,15 @@ func (a *WSAsyncClientPool) DisconnectAll() {
 	}
 }
 
-func (a *WSAsyncClientPool) AddEventHandler(h WSClientEventHandler, namespaces []string) {
+func (a *WSMessageAsyncClientPool) AddEventHandler(h WSClientEventHandler) {
 	a.eventHandlersLock.Lock()
 	a.eventHandlers = append(a.eventHandlers, h)
-
-	// add this handler per namespace
-	for _, ns := range namespaces {
-		if _, ok := a.nsEventHandlers[ns]; !ok {
-			a.nsEventHandlers[ns] = []WSClientEventHandler{h}
-		} else {
-			a.nsEventHandlers[ns] = append(a.nsEventHandlers[ns], h)
```

Comment:
I think you broke the wildcard namespace by removing, basically you forgot to set it afterwards